test/python/common.py

```diff
@@ -367,13 +370,13 @@ def _wrapper(self, *args, **kwargs):
         qiskit_wrapper._DEFAULT_PROVIDER = DefaultQISKitProvider()
         kwargs.update(_get_credentials(self, TEST_OPTIONS))
 
+        decorated_item = func
```

Comment:
I forgot to change `decorated_item` with `decorated_func`. Do you mind to do it for me?

Reply:
88be95c